testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -9319,6 +9323,7 @@ fn skip_mining_long_tx() {
     naka_conf.node.prometheus_bind = Some(prom_bind.clone());
     naka_conf.miner.wait_on_interim_blocks = Duration::from_secs(1);
     naka_conf.miner.nakamoto_attempt_time_ms = 5_000;
+    naka_conf.miner.tenure_cost_limit_per_block_percentage = Some(100);
```

Comment:
Won't this cause an error when parsing the config?

Reply:
No it won't break the test because it doesn't go through the miner config. But you are right that I should update that check to allow 100.

Reply:
oh yeah, right. thanks!